src/install/config/java.yaml

```diff
@@ -72,6 +72,8 @@ appInfo:
       - value: 'aws'
         displayName: 'AWS Lambda'
         logo: 'awsLambda'
+      - value: 'ansible'
```

Comment:
something is missing here*** AWS lambda immediately shortens and keeps the simple 2-step structure (a la VM). ask shawn about maneuvering this around

Reply:
I just noticed this comment @akristen. Still need help with this?

Reply:
No! We talked about it. I need to add value: ansible where the value: aws appears. :) Thanks for checking in!